### PR TITLE
Upload list content item

### PR DESCRIPTION
**Pull request by Noy-Maimon**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/39603

## Description
Within the effort of adding lists to be as content items, this PR adds the support for that in the upload flow to the MP.

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

**Comment by Noy-Maimon** [comment]
Successful build with RN aggregation of two lists (under the pack `Barracuda`): https://app.circleci.com/pipelines/github/demisto/content/109211/workflows/c3be6abf-2a05-4273-8f1c-799a8be368b7/jobs/415376

File: Tests/Marketplace/marketplace_services.py [hunk 1]
```diff
@@ -1476,7 +1476,8 @@ def collect_content_items(self):
                 PackFolders.GENERIC_DEFINITIONS.value: "GenericDefinitions",
                 PackFolders.GENERIC_FIELDS.value: "GenericFields",
                 PackFolders.GENERIC_MODULES.value: "GenericModules",
-                PackFolders.GENERIC_TYPES.value: "GenericTypes"
+                PackFolders.GENERIC_TYPES.value: "GenericTypes",
+                PackFolders.LISTS.value: "list"
```
Comment (guyfreund) [inline Tests/Marketplace/marketplace_services.py:1480]: are we sure it's not capitalized?

File: Tests/Marketplace/marketplace_services.py [hunk 2]
```diff
@@ -1616,6 +1617,10 @@ def collect_content_items(self):
                             'dataType': content_item.get('dataType', ""),
                             'widgetType': content_item.get('widgetType', "")
                         })
+                    elif current_directory == PackFolders.LISTS.value:
+                        folder_collected_items.append({
+                            'name': content_item.get('name', "")
+                        })
```
Comment (guyfreund) [inline Tests/Marketplace/marketplace_services.py:1623]: what about description?

File: Tests/Marketplace/marketplace_services.py (hunk 1 above)
Comment (Noy-Maimon) [reply Tests/Marketplace/marketplace_services.py:1480]: singulre and lowercase as can be seen here: 
https://github.com/demisto/server/blob/6a8074b45de1f8ccc53cd774dc02a9a1d6e95623/domain/contentpack.go#L224

File: Tests/Marketplace/marketplace_services.py (hunk 2 above)
Comment (Noy-Maimon) [reply Tests/Marketplace/marketplace_services.py:1623]: Only name as can be seen here: https://github.com/demisto/server/blob/6a8074b45de1f8ccc53cd774dc02a9a1d6e95623/domain/contentpack.go#L343

**Review by guyfreund — APPROVED** [review]

**Pull request merged**